### PR TITLE
feat: replat-6101 edition tile summary padding fix

### PR DESCRIPTION
**Pull request by parkjh**
- removes unnecessary padding of empty element when there are no expiable flags prop set

### CODE REVIEW

**Comment by parkjh** [comment]
Screenshot 2019-04-29 at 14 45 04

**Comment by parkjh** [comment]
Screenshot 2019-04-29 at 15 26 23

File: packages/section/__tests__/shared-tracking.js [hunk 1]
```diff
@@ -74,6 +74,35 @@ export default () => {
...
+    link.props.onPress();
+
+    const [[call]] = stream.mock.calls;
+
+    expect(call).toMatchSnapshot();
+    expect(onPuzzlePress.mock.calls).toMatchSnapshot("onPuzzlePress");
+  });
+};
+
+// https://nidigitalsolutions.jira.com/browse/REPLAT-6294
```
Comment (aronshamash) [inline packages/section/__tests__/shared-tracking.js]: We don't really need the jira reference

File: packages/edition-slices/src/tiles/shared/tile-summary.js [hunk 2]
```diff
@@ -37,7 +37,10 @@ class TileSummary extends Component {
       starStyle
     } = this.props;
 
-    if (!withStar) {
+    if (!withStar || Platform.OS === "ios") {
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/shared/tile-summary.js]: ideally we wouldn't have a `Platform.OS` test but use a `.android.js` and `.web.js` file (`.js` is the picked up by iOS)

File: packages/edition-slices/src/slices/shared/gutter.js [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
-  >
-    {children}
-  </View>
+  <ResponsiveContext.Consumer>
+    {({ isTablet }) => (
+      <View
+        style={{
+          alignSelf: "center",
+          maxWidth: "100%",
+          paddingHorizontal: isTablet ? spacing(2) : spacing(0),
```
Comment (aronshamash) [inline packages/edition-slices/src/slices/shared/gutter.js]: `spacing(2) : 0` is fine

File: packages/edition-slices/src/slices/shared/gutter.js [hunk 4]
```diff
@@ -1,18 +1,24 @@
...
-      width: editionMaxWidth
-    }}
-  >
-    {children}
-  </View>
+  <ResponsiveContext.Consumer>
+    {({ isTablet }) => (
+      <View
+        style={{
+          alignSelf: "center",
```
Comment (ValGeorgiev) [inline packages/edition-slices/src/slices/shared/gutter.js]: Please don't use "inline" styling, you can move this in a styling file and use breakpoints there

Comment (parkjh) [reply packages/edition-slices/src/slices/shared/gutter.js]: @ValGeorgiev that styling was already there, I thought best not to change it

File: packages/section/__tests__/shared-tracking.js (hunk 1 above)
Comment (parkjh) [reply packages/section/__tests__/shared-tracking.js]: forgot this was public repo...

File: packages/edition-slices/src/tiles/shared/tile-summary.js (hunk 2 above)
Comment (parkjh) [reply packages/edition-slices/src/tiles/shared/tile-summary.js]: @aronshamash happy to chat about this after standup?

Comment (parkjh) [reply packages/edition-slices/src/tiles/shared/tile-summary.js]: so the tile star should not be rendered to iOS at all

File: packages/edition-slices/src/slices/shared/gutter.js (hunk 4 above)
Comment (ValGeorgiev) [reply packages/edition-slices/src/slices/shared/gutter.js]: Hm ok, we will have a ticket for Gutter refactoring and we can leave it for now

**Pull request merged**